### DISCUSSION TITLE
Modyfikacja zasobu API "zwroty klienckie" w kontekście ID lub login klienta

### DISCUSSION THREAD

**Discussion by OnesAndZerosPL** [allegro/allegro-api#5504] [Suggestions - resolved]
Zwracam się z uprzejmną prośbą o rozważenie możliwości rozszerzenia implementacji zasobu "/order/customer-returns" o następujace funkcje:
- zwracanie login ID klienta dokonującego zwrotu (takie informacje są przekazywane w ramach email do sprzedawcy ale nie w Api)
- inny identyfikator klienta np. email (może być zaanonimizowany przez allegro)
- możliwość filtru po danym użytkowniku (Client ID)

Powyższe funkcje w znaczy sposób wpłyną na możliwość automatyzacji procesu zwrotów przez integracje (np. w naszym przypadku sklep IDOSell). Brakuje tych informacji zwrotnych, co uniemożliwia pełną identyfikację sprzedazy.

**Comment by jakubmaguza** [comment]
Na razie, dopóki tego nie wprowadzą, możecie robić "na około" ->
W zwrotach macie informację o orderID. Na jego podstawie możecie albo we własnej bazie wyszukać po orderID konkretne zamówienie i dane klienta, a jeśli danego zamówienia w bazie nie macie, to odpytać API allegro o zamówienie o konkretnym ID.

**Comment by MartaNowaczyk** [comment]
**UPDATE:** Dzięki tej sugestii od marca proponowane zmiany [są dostępne w zwrotach klienckich](https://developer.allegro.pl/news/zwroty-klienckie-dodalismy-nowe-pola-Pgb8aLz6wfE). Zmieniłam status na "Suggestions - resolved".
